source/renderer/app/components/voting/VotingUnavailable.js

```diff
@@ -1,132 +1,30 @@
 // @flow
```

Comment:
Since the voting registration is always open now, the voting closed logic was removed.